static/data/team.yml

```diff
@@ -2,14 +2,14 @@
   people:
     - name: Matteo Collina
       sortname: Collina Matteo
-      picture: 'https://avatars1.githubusercontent.com/u/52195?v=4&s=460'
+      githubId: '52195'
```

Comment:
it does not seem this change help us IMHO

before:

- copy link
- paste link

now:

- ask yourself: where should I get this id?
- copy link
- paste link
- edit link

Reply:
https://caius.github.io/github_id/#uzlopak

Reply:
Could you add a comment upper in this file?

Reply:
is it now good?